### ISSUE TITLE
Replay UX improvements

### ISSUE THREAD

**Issue by daibhin** [PostHog/posthog#21302]
As part of https://github.com/PostHog/posthog/issues/21078 Team Replay will be working on the UX on Session Replay over the next quarter. This is a tracking issue for specific UX improvements & UI changes we plan on making

### Current UI
At its best
<img width="1169" alt="Screenshot 2024-04-03 at 14 58 00" src="https://github.com/PostHog/posthog/assets/6685876/7e9e5a17-830b-4f53-a306-06df7fdc9b7b">

At its worst
<img width="1841" alt="Screenshot 2024-04-03 at 14 59 08" src="https://github.com/PostHog/posthog/assets/6685876/90fb8612-c7f3-47e3-883b-9d18c23ebb02">

### Feedback

General UI busyness
> Currently I find this UI overwhelming even without the sidebar, and it feels like we’re only adding more and more functionality and UI complexity to replays. I wonder if it would be worth so that, when someone goes to /replays, we don’t show them the full list of replays and start automatically playing the latest video. Instead we show them a page for customizing filters first so they can choose what to engage with more decidedly?
Plus we’re repeating info in a few places (e.g. we have Chrome Desktop Windows above the recording video, but it’s also highlighted in the list already). Feels like somewhere we could remove UI elements.

Quick vs Advanced filters
> - It wasn’t obvious that the “Choose quick filter” option wasn’t the actual filter options. I went to filter by email, but that just adds it as a quick filter option, which you then need to select + configure. I didn’t immediately get this was the intention.
It also isn’t obvious that button is a dropdown at all as it doesn’t look like any other dropdown button.
> - Overall, it doesn’t feel substantially quicker / easier to add filters this way. The only step it skips is searching for the property. I feel like it would be easier / better if the search modal just automatically highlighted popular filters immediately, rather than setting quick filters separately.
> - Clicking on Action or event in advanced filters automatically applies the pageview event, which means I have to click again to search for a different action or event. Just eliminating this would probably remove the need for these separate quick filters. The Person or cohort button doesn’t do this, which is good.
> - If you want a “quick option”, I’d prefer it if I could configure a filter using the “advanced” options and then choose to save that config for later use, if that makes sense? That way, instead of this “Choose quick filter” button you can just have a list/dropdown of previously saved filters, and creating new ones means using the advanced filters to configure them.
> TL;DR, my preference as a user would be to just to make the advanced filters easier to use, rather than adding more UI options for quick filters

### Related issues
[Saved filters](https://github.com/PostHog/posthog/issues/20314)

**Comment by daibhin** [comment]
- [ ] Container query on inspector sidebar
- [ ] Tertiary button to change time on on bottom left hand of player
- [ ] Playlists as tabs with overflow inside a dropdown button
  - [ ] Include default tabs for "All sessions" (rename "Recent") and "Errors"
  - [ ] Move playlist creation to be inline
  - [ ] Allow updating of playlist filters without immediately saving
- [ ] Move "Playback from file" to header under ellipse
- [ ] Collapsible user list on left hand side
- [ ] Simplify user sidebar info (with extra in hover popover)
- [ ] Move user info into the right hand panel
- [ ] Full width bottom bar (move left and right columns into the same level as the player)
- [ ] Make inspector timestamp and sync scrolling buttons clearer

Stretch
- [ ] Table view of recordings
- [ ] Show next / previous recordings for a user

**Comment by daibhin** [comment]
Achieved everything we needed to here last quarter. We now have a new issue with any outstanding improvements: https://github.com/PostHog/posthog/issues/24561
